auto-instrumentation/okhttp/okhttp-3.0/README.md

```diff
@@ -0,0 +1,41 @@
+# Android Instrumentation for OkHttp version 3.0 and higher
+
+Provides OpenTelemetry instrumentation for [okhttp3](https://square.github.io/okhttp/).
+
+## Quickstart
+
+### Add these dependencies to your project
+
+Replace `OPENTELEMETRY_VERSION` with the [latest
+release](https://search.maven.org/search?q=g:io.opentelemetry.instrumentation%20AND%20a:opentelemetry-okhttp-3.0).
+
+Replace `BYTEBUDDY_VERSION` with the [latest
+release](https://search.maven.org/search?q=g:net.bytebuddy%20AND%20a:byte-buddy).
```

Comment:
> shouldn't the maven search be for the io.opentelemetry.android coordinates

Yeah 😅 I forgot that part when copy/pasting from the instrumentation repo.

Reply:
> We don't have to change it for this PR, but I think users would benefit from just having the real values in here.

I'd love to show the versions there automatically instead of having to use those maven links!